crates/subspace-node/src/bin/subspace-node.rs

```diff
@@ -446,6 +454,23 @@ fn main() -> Result<(), Error> {
                     secondary_chain_node.network_starter.start_network();
                 }
 
+                // TODO: Workaround for regression in Gemini 1b 2022-jun-08 release:
+                //  we need to reset network identity of the node to remove it from block list of
+                //  other nodes on the network
```

Comment:
I'm wondering how the block list is related, could you expand a little bit on the context?

Reply:
Yes, basically the bug was with block height where we check for solution range. I forgot that solution range in a particular block is taken from previous block, so nodes get stuck at this block.

Additionally, node gets banned by other nodes in the process, which is why if node is stuck at this block we need to reset network identity and start again.

Reply:
Hmm, is it because some nodes submitted a vote which contains a solution in the wrong solution range, and then it's added to the block list in pallet-subspace and gets banned by others in the end? I thought the content in the block list is the farmer identity, then why reset the node identity? I might have some misunderstanding here, please correct me :D

Reply:
Still not super straightforward to see the whole background, but the block list in the networking layer makes more sense.